apps/console/src/extensions/configs/models/documentation.ts

```diff
@@ -462,6 +468,8 @@ export interface DocumentationLinksExtensionInterface {
          * Documentation links for account recovery elements.
          */
         accountRecovery: AccountRecoveryDocumentationLinksInterface;
+
```

Comment:
Let's add a Doc comment to maintain consistency

Reply:
Fixed